29ki/29k
# Move public host check inside create session flow

Issue: https://www.notion.so/29k/Closed-BETA-0055aabd797b40c7b390b7cea4eba658#10d6f31782a34cd38a0bf69ced519684

### Description of the Change



#### Screenshots


| Before | After |
|-|-|
| ![Screenshot_1667236896](https://user-images.githubusercontent.com/7523828/199069815-ec1fc687-3357-40a7-8a89-e90c7c74cfa1.png) |  ![Screenshot_1667241872](https://user-images.githubusercontent.com/7523828/199085388-85acf98b-9b7d-4670-8015-31517995b81d.png) |

"Pick flow" screen (UX / UI Needed, will point it out [in notion and ask for help](https://www.notion.so/29k/Closed-BETA-0055aabd797b40c7b390b7cea4eba658#520dd6b731cf49d4b1f9ce9db7b8b5be))
<img width="300" src="https://user-images.githubusercontent.com/7523828/199085636-c3e36242-9262-4026-a62b-34a36a78aa24.png" />


And normal users get `private` session type by behind the scenes:
https://github.com/29ki/29k/blob/088bfb0ff117a5c0a8c5550813a0a7be4667a0e7/client/src/routes/Sessions/components/CreateSessionModal.tsx#L269-L271

### Verification Process

- Use the app as a anonymous user, see if you can create a private session only.
- Use the app as a host, see if you can choose between public and private sessions on the creation flow.
